src/main/java/com/cburch/logisim/std/gates/GateKeyboardModifier.java

```diff
@@ -12,75 +12,77 @@
 import com.cburch.logisim.comp.Component;
 import com.cburch.logisim.data.AttributeSet;
 import com.cburch.logisim.gui.main.Canvas;
+import com.cburch.logisim.prefs.AppPreferences;
+import com.cburch.logisim.prefs.PrefMonitorKeyStroke;
 import com.cburch.logisim.tools.SetAttributeAction;
+
+import java.awt.event.InputEvent;
 import java.awt.event.KeyEvent;
 
 public class GateKeyboardModifier {
   public static boolean tookKeyboardStrokes(
       int KeyCode,
+      int modifier,
       Component comp,
       AttributeSet attrs,
       Canvas canvas,
       SetAttributeAction act,
       boolean CreateAction) {
-    switch (KeyCode) {
-      case KeyEvent.VK_N:
-      case KeyEvent.VK_S:
-        if (attrs.containsAttribute(GateAttributes.ATTR_SIZE)) {
-          if (CreateAction) act.set(comp, GateAttributes.ATTR_SIZE, GateAttributes.SIZE_NARROW);
-          else {
-            attrs.setValue(GateAttributes.ATTR_SIZE, GateAttributes.SIZE_NARROW);
-            canvas.repaint();
-          }
+    String compare = InputEvent.getModifiersExText(modifier) + " + " + KeyEvent.getKeyText(KeyCode);
+    /* Here originally have VK_N VK_EQUALS VK_PLUS VK_MINUS but not-used */
+    if (compare.equals(compare.equals(((PrefMonitorKeyStroke) AppPreferences.HOTKEY_GATE_MODIFIER_SIZE_SMALL).getCompareString()))) {
+      if (attrs.containsAttribute(GateAttributes.ATTR_SIZE)) {
+        if (CreateAction) act.set(comp, GateAttributes.ATTR_SIZE, GateAttributes.SIZE_NARROW);
```

Comment:
Use curly brackets and put the subordinate statement on a different line.

I know that the original was the way it is here. We are working at modifying our code to meet google java coding standards so we are trying to fix things that we touch. The standard is to always use the curly brackets and put subordinate code on a new line. You need not fix the entire file(s), but we would appreciate your doing part you modify.

Do this in other places where you modified or added code.

Your code in new files looked good.